### PR TITLE
 BUGFIX: TMA-1528: remove CollectSegments and CollectDataProduct for add and remove in users brick

### PR DESCRIPTION
**Pull request by tqtu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/309/ : SUCCESS in 37s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/310/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/311/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/312/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/313/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/314/ : SUCCESS in 7s

File: lib/gooddata/lcm/lcm2.rb [hunk 1]
```diff
@@ -305,6 +305,12 @@ def perform(mode, params = {})
           skip_actions.include?(action.name.split('::').last)
         end
 
+        sync_mode = params.fetch(:sync_mode, nil)
+        if mode == 'users' && %w[add_to_organization remove_from_organization].include?(sync_mode)
```
Comment (phong-nguyen-duy) [inline lib/gooddata/lcm/lcm2.rb]: how about user_filter? does it have these modes and do we need to exclude as well?

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/424/ : SUCCESS in 14s

**Pull request merged**